### PR TITLE
update apt module

### PR DESCRIPTION
**Pull request by cgzones**
* apt runs fine in confined mode, dpkg is the heavy one

### CODE REVIEW

File: apt.te [hunk 1]
```diff
@@ -25,22 +25,25 @@ files_tmp_file(apt_tmp_t)
...
 
-type apt_var_lib_t alias var_lib_apt_t;
+type apt_var_lib_t;
 files_type(apt_var_lib_t)
 
 type apt_var_log_t;
 logging_log_file(apt_var_log_t)
 
+type apt_unit_t;
+init_unit_file(apt_unit_t)
```
Comment (pebenito) [inline apt.te]: It doesn't make sense for a package manager to be a daemon.

File: apt.te [hunk 2]
```diff
@@ -149,7 +152,3 @@ optional_policy(`
 	rpm_read_db(apt_t)
 	rpm_domtrans(apt_t)
 ')
-
-optional_policy(`
-	unconfined_domain(apt_t)
```
Comment (pebenito) [inline apt.te]: Please do not remove.

File: apt.te (hunk 1 above)
Comment (cgzones) [inline apt.te]: apt has a systemd timer, apt-daily, which can update the package database and download updated packages

Comment (pebenito) [inline apt.te]: Is it possible to split the networking portion out?

**Comment by cgzones** [comment]
closing for now, needs some more work

**Pull request closed without merging**